### PR TITLE
ENH:Function segmentation

### LINKED ISSUE #2: Function segmentation
하나의 기능만을 수행하는 함수로 세분화하기 위해서, 
checkforwin()함수를 가로,세로,대각선, 비긴상태를 확인하기 위해서 각각  checkforHorizontal, checkforVertical, checkforDiagonal, checkforDraw,의 함수로 세분화한다.

### PR DESCRIPTION
**Pull request by mjseok**
The checkforwin function is subdivided into checkHorizontal,checkVertical,checkDiagonal,checkDraw.

The checkHorizontal function checks the completion of the horizontal line.
The checkVertical function checks the completion of the vertical line.
The checkDiagonal function checks the completion of the diagonal line.
The checkDraw function checks if all the fields are filled.

### CODE REVIEW

File: Code.c [hunk 1]
```diff
@@ -28,10 +28,10 @@ int main()
...
     read:
         p=fopen("score.txt","a+");
-    printf("\nEnter name of player1: ");
-    scanf("%s",u1);
-    fprintf(p,"\n%s",u1);
-    printf("Enter name of player2: ");
+        printf("\nEnter name of player1: ");
+   	scanf("%s",u1);
+   	fprintf(p,"\n%s",u1);
+    	printf("Enter name of player2: ");
```
Comment (mjseok) [inline Code.c:34]: 들여쓰기 고치기

**Comment by cherry2030** [comment]
main 부분도 수정해야할것 같아요...!

**Comment by hanzzb** [comment]
좋습니다!

**Comment by seohyeonKK** [comment]
실행이 안되는데 다시 한 번 확인 부탁드려요~

**Comment by hanzzb** [comment]
main만 고치고 merge합시다!!!

**Comment by cherry2030** [comment]
main 변경했으니까 이제 merge합시당~~

**Pull request merged**